security/cert_manager_operator/cert-manager-operator-issuer-acme.adoc

```diff
@@ -6,7 +6,29 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-The {cert-manager-operator} supports using ACME CA servers, such as Let's Encrypt, to issue certificates.
+The {cert-manager-operator} supports using Automated Certificate Management Environment (ACME) CA servers, such as Let's Encrypt, to issue certificates. Explicit credentials are configured by specifying the secret details in the `Issuer` API object. Ambient credentials are extracted from the environment, metadata services, or local files which are not explicitly configured in the `Issuer` API object.
+
+[NOTE]
+====
+`Issuer` can only issue certificates from a specific namespace. You can also use `ClusterIssuer` to solve ACME DNS-01 challenges which can issue certificates across all namespaces in the cluster.
```

Comment:
ClusterIssuer can issue certificates for all namespaces, this does not only apply to ACME clusterissuer, but also for all types of clusterissuers, so removing specific words "solve ACME DNS-01 challenges which can".
```suggestion
`Issuer` is namespace scoped, it can only issue certificates of the same namespace. You can also use `ClusterIssuer` to issue certificates across all namespaces in the cluster.
```

Reply:
Addressed